src/code/z_player_lib.c

```diff
@@ -1367,7 +1367,7 @@ u32 func_80091738(GlobalContext* globalCtx, u8* segment, SkelAnime* skelAnime) {
     SkelAnime_InitLink(globalCtx, skelAnime, gPlayerSkelHeaders[(void)0, gSaveContext.linkAge], &gPlayerAnim_003238, 9,
                        ptr, ptr, PLAYER_LIMB_MAX);
 
-    return size + 0x8890;
```

Comment:
Is there a reason for this?

Reply:
its a fake return. asm is using v0 for this but it doesnt actually return or use this value

Reply:
(deleted the last comment cause I misunderstood)
but yea, the base is `segment + 0x8800` seen in the code above. and its returning whatever is here + 0x10

Reply:
The 0x8800 is for the `segment + 0x8800` while the 0x90 is the size of the player joint table, so that's why I split them, tho maybe we should use `PLAYER_LIMB_BUF_COUNT * sizeof(Vec3s)` instead of 0x90.

Note that this function does a lot of hardcoded stuff already. The worst part is probably how it loads the whole gameplay_keep object at `segment + 0x3800` but only keeps 0x5000 bytes of it since it starts loading other stuff at `segment + 0x8800`, and I guess it works because those 0x5000 bytes contain everything it needs (shields and maybe something else?)